microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -1,70 +1,88 @@
 
-Microsoft recommends that you use [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] whenever possible for the following reasons.  
+This article describes the methods Microsoft Edge supports for retrieving user agent information.
+
+## Feature detection
+
+Microsoft recommends [detecting if a feature is supported][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] in your browser whenever possible instead of detecting the browser.
+
+If you must detect browsers, Microsoft recommends using client hints as follows.
+
+## User-Agent Client Hints
+
+Microsoft Edge supports User-Agent Client Hints starting with version 90.
 
-*   Improve code maintainability.  
-*   Reduce code fragility.  
-*   Reduce code breakage from changes to the User-Agent string.  
-    
-When [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] isn't applicable and you must use user agent detection, use the following format to detect Microsoft Edge user agent on Windows and Android.  
+User-Agent Client Hints is a cleaner, more privacy-preserving way to access browser information such as browser name, version number, platform, and more. Soon the User-Agent string will be frozen and deprecated by various browsers. For example, the Chrome Platform Status site describes the change in [Feature: Reduce User-Agent string information][https://www.chromestatus.com/feature/5704553745874944].
 
-## User-Agent Client Hints  
+Employ User-Agent Client Hints when you want to:
+- Determine if new browser activity is from the expected user.
+- Customize hints or instructions if the user is new to this site.
 
-Starting in Microsoft Edge version 90, access browser information in a cleaner, more privacy-preserving way.  
+Do not use User-Agent Client Hints to:
+- Block *unsupported* browsers.
+- Restrict access to features on your site.
 
-### User-Agent Client Hints HTTPS Header  
+For more information, navigate to the at [W3C Community Draft Report | User-Agent Client Hints][https://wicg.github.io/ua-client-hints/].
 
-By default, Chromium browsers including Microsoft Edge send the `Accept-CH-UA` response header in the following format.  
+### User-Agent Client Hints HTTPS header
+
+When Microsoft Edge sends an HTTPS request to a server, it sends the User-Agent Client Hints headers that correspond to browser brand and mobile information. If the server requires more granular information about the browser, its response includes an `Accept-CH` header. The value of that response header is a comma-separated list of all the Client Hints request headers the server wants from the browser, such as `Accept-CH: Sec-CH-UA-Platform,Sec-CH-UA-Platform-Version`. The next Microsoft Edge HTTPS request to the server will include the requested User-Agent Client Hints headers.
+
+By default, Chromium browsers including Microsoft Edge send the `Sec-CH-UA` and `Sec-CH-UA-Mobile` response header in the following format.  

```

Comment:
I learned recently that the Brave browser, another Chromium-based browser, isn't intending to support UA client hints. For that reason, I'm thinking we should stick to talking about Edge only in this part. Ex:

> By default, Microsoft Edge sends the

Also, should we use a colon instead of a period at the end of the sentence?

Reply:
https://docs.microsoft.com/en-us/style-guide/punctuation/colons

Reply:
Use colons sparingly ... I'm using that to justify not changing this one ... but it might be a gray area.  Otherwise fixed.